content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,609 @@
+    ```
+
+    See how both Kyverno `ClusterPolicies` have been applied and resulted in all the other default values being inserted for us!
+
+1. To be absolutely sure we have not enforced any settings, let us try and set every value of the `Certificate` explicitly, that we have a default rule for. We will use a final `Certificate` named "test-revision-override":
+
+    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/cert-test-revision-override.yaml
+    ```
+    🔗 <a href="cert-test-revision-override.yaml">`cert-test-revision-override.yaml`</a>
+
+1. Dry-run apply and `diff` this file:
+
+    ```shell
+    kubectl apply -f cert-test-revision-override.yaml --dry-run=server -o yaml | diff -uZ cert-test-revision-override.yaml -
+    ```
+
+    This should return:
+
+    ```yaml
+    --- cert-test-revision-override.yaml	2023-09-07 16:58:41.363648093 +0100
+    +++ -	2023-09-08 11:09:16.026674875 +0100
+    @@ -1,8 +1,14 @@
+    apiVersion: cert-manager.io/v1
+    kind: Certificate
+    metadata:
+    +  annotations:
+    +    kubectl.kubernetes.io/last-applied-configuration: |
+    +      {"apiVersion":"cert-manager.io/v1","kind":"Certificate","metadata":{"annotations":{},"name":"test-revision-override","namespace":"default"},"spec":{"dnsNames":["example.com"],"issuerRef":{"group":"cert-manager.io","kind":"ClusterIssuer","name":"not-my-corp-issuer"},"privateKey":{"algorithm":"RSA","encoding":"PKCS8","rotationPolicy":"Never","size":4096},"revisionHistoryLimit":44,"secretName":"test-revision-override-cert"}}
+    +  creationTimestamp: "2023-09-08T10:09:16Z"
+    +  generation: 1
+      name: test-revision-override
+      namespace: default
+    +  uid: f9269c72-828c-4647-988c-1fee1e38afe0
+    spec:
+      dnsNames:
+      - example.com
+    ```
+
+    From this command you can see that, none of the `Certificate` specification fields have been changed.
+    Only the metadata section has changed which tells us the policies have applied but not set any defaults because values were already provided. 
+    So you maintain the flexibility to override the cluster defaults when needed.
+
+## 3 - Defaulting through Ingress Annotations
+
+Many cert-manager users don't create `Certificate` resources directly and instead use the [ingress-shim](https://cert-manager.io/docs/usage/ingress/) functionality.
+cert-manager creates `Certificate` resources based on the [supported annotations](https://cert-manager.io/docs/usage/ingress/#supported-annotations) and the `Ingress` specification.
+Let's see how we can still use `ClusterPolicy` to apply our defaults in this use case.
+
+1. Check the example `Ingress` resource has the correct annotation to select an `Issuer` or `ClusterIssuer`:
+
+    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/ingress.yaml
+    ```
+    🔗 <a href="ingress.yaml">`ingress.yaml`</a>
+
+1. This annotation and the relevant `ingress.spec.tls` configuration is all we need:
```

Comment:
```suggestion
1. This annotation and the relevant `ingress.spec.tls` configuration are all we need:
```

Reply:
Little unsure here. I would agree `configurations are` when the first word is plural.
Unless the assumption is that `configuration` is a plural term, in that there are multiple fields in `ingress.spec.tls`, which is true,.
I've changed it as you suggest for now.

Reply:
I'm reading it as "(annotation + configuration) are".

Reply:
That makes sense, I was too zoomed in on the end of the sentence. :)